src/renderer/pages/project/project.vue

```
@@ -519,34 +522,65 @@
 const imageFiles = ref<(ArrayBuffer | null)[]>([]);
 const movieFiles = ref<(ArrayBuffer | null)[]>([]);
 
+const arrayPositionUp = <T,>(dataSet: T[], index: number) => {
+  const newData = [...dataSet];
+  const temp = newData[index - 1];
+  newData[index - 1] = newData[index];
+  newData[index] = temp;
+  return newData;
+};
+const arrayAdd = <T,>(dataSet: T[], index: number, data: T) => {
```

ESLint

[error] 'data' is defined but never used. Allowed unused args must match /^__/u.

(@typescript-eslint/no-unused-vars)